src/status_im2/contexts/wallet/account/view.cljs

```diff
@@ -10,6 +10,34 @@
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(defn account-options
+  []
+  [rn/view
+   [quo/drawer-top temp/account-data]
+   [quo/action-drawer
+    [[{:icon                :i/edit
+       :accessibility-label :edit
+       :label               (i18n/label :t/edit-account)}
+      {:icon                :i/copy
+       :accessibility-label :copy-address
+       :label               (i18n/label :t/copy-address)}
+      {:icon                :i/share
+       :accessibility-label :share-account
+       :label               (i18n/label :t/share-account)}
+      {:icon                :i/delete
+       :accessibility-label :remove-account
+       :label               (i18n/label :t/remove-account)
+       :danger?             true}]]]
+   [quo/divider-line]
+   [rn/view
+    {:style {:padding-horizontal 20
+             :padding-top        12
+             :padding-bottom     8}}
+    [quo/section-label {:section (i18n/label :t/select-another-account)}]]
+   [rn/view {:style {:margin-horizontal 8}}
+    (for [account temp/other-accounts]
+      [quo/account-item {:account-props account}])]])
```

Comment:
This might produce a unique key warning. We can set the account public key as the key here.

Additionally, we can move this to FlatList. WDYT?